src/components/logo.js

```diff
@@ -23,7 +19,7 @@ const Logo = ({ color }) => {
-          fill={colorCode}
+          fill="currentColor"
         />
         <defs>
           <radialGradient
```

Comment:
IE and Edge have a few quirks with `currentColor` on gradients. Can you perhaps test this there? Just to confirm.

Reply:
It does not work at all in IE 11, but the website is completely screwed in there anyway, so I wouldn't bother with it.

It works just fine on Edge.

Tested with https://deploy-preview-230--subvisual.netlify.com/.

![2020-02-21_13-18](https://user-images.githubusercontent.com/1141870/75037832-fc2ae880-54ac-11ea-99f7-0c000fed728d.png)
![2020-02-21_13-20](https://user-images.githubusercontent.com/1141870/75037837-fe8d4280-54ac-11ea-859b-6fccebe6ad35.png)

Reply:
Well, if IE 11 is not a thing, go for it :+1: